### PR TITLE
Add reminder of fluid.layers.lstm

### PR DESCRIPTION
**Pull request by GaoWei8**
### PR types
Others

### PR changes
APIs

### Describe
增加fluid.layers.lstm已经废弃的描述，并给出新接口和说明。

 
### 需要approve
```
2020-11-23 10:58:32 0. APIs without core.ops: 
2020-11-23 10:58:32 paddle.fluid.layers.lstm
2020-11-23 10:58:32 You must have one RD (zhiqiu (Recommend) or phlrain) approval for the api change for the opreator-related api without 'core.ops'.
2020-11-23 10:58:32 For more details, please click [https://github.com/PaddlePaddle/Paddle/wiki/paddle_api_development_manual.md]
2020-11-23 10:58:32 
2020-11-23 10:58:32 There are 1 approved errors.
```

### CODE REVIEW

**Review by wangchaochaohu — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/rnn.py [hunk 1]
```diff
@@ -2543,6 +2543,11 @@ def lstm(input,
     weight_size = 0
     num_dirrection = 2 if is_bidirec == True else 1
 
+    print(
+        'This API (fluid.layers.LSTM) has been deprecated. It is recommended to use the new API (class paddle.nn.LSTM).'
+        'Please refer to the link: https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/layer/rnn/LSTM_cn.html#lstm'
+    )
+
```
Comment (zhiqiu) [inline python/paddle/fluid/layers/rnn.py]: Why not use @deprecated decorator?

**Review by zhiqiu — APPROVED** [review]
LGTM, it is used in  static graph.

File: python/paddle/fluid/layers/rnn.py (hunk 1 above)
Comment (GaoWei8) [reply python/paddle/fluid/layers/rnn.py]: done

**Review by wangchaochaohu — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**